vendor/blanket-require.js

```diff
@@ -50,6 +50,10 @@ var shouldExclude = function(moduleName) {
       return true;
     }
 
+    if ( moduleName === blanket.options('modulePrefix') ) {
```

Comment:
the only question I have is what happens when I actually have an 'index' file 

I need to look for an example but believe it's possible